app/controllers/api/submissions_controller.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+class Api::SubmissionsController < ApplicationController
+  before_action :set_question, only: %i[question]
+
+  def question
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Metrics/AbcSize: Assignment Branch Condition size for question is too high. [34.37/15]